argoverse/sensor_dataset_config.py

```diff
@@ -0,0 +1,41 @@
+from typing import NamedTuple, Optional
+
+from hydra.experimental import compose, initialize_config_module
+from hydra.utils import instantiate
+
+
+class ImageDimensions(NamedTuple):
+    """Dimensions are provided in pixels."""
+
+    height: int
+    width: int
+
+
+class SensorDimensions(NamedTuple):
```

Comment:
For configuration classes, what do you think about maintained the `Config` suffix -- e.g., `SensorDimensionsConfig`? I've seen that consistently used in other repositories related to Pytorch.

Reply:
I like the `Config` suffix -- should be incorporated now